editor/editor_properties.cpp

```diff
@@ -1751,7 +1751,7 @@ void EditorPropertyVector2::setup(double p_min, double p_max, double p_step, boo
 	if (!p_link) {
 		linked->hide();
 	} else {
-		linked->set_pressed(true);
+		linked->set_pressed(EDITOR_GET("interface/inspector/enable_vector_property_linking_by_default"));
```

Comment:
Should this be cached to avoid fetching editor settings multiple times every time the inspector refreshes?

Reply:
I'll look into it, but editor properties are instantiated each time, so it needs to be cached somewhere in the inspector itself.

Reply:
Although things like `"interface/inspector/horizontal_vector2_editing"` don't seem to be cached 🤔